.eslintrc.js

```diff
@@ -132,7 +132,7 @@ module.exports = {
     'max-classes-per-file': 'error',
     'no-lonely-if': 'error',
     'no-unneeded-ternary': 'error',
-    'no-use-before-define': 'error',
+    'no-use-before-define': 'off',
```

Comment:
This doesn't seem impactful to what we're doing, and was making readability tough in the codebase.